### PR TITLE
usage: reword build

### PR DESCRIPTION
**Pull request by leongross**
See PR [#4467](https://github.com/tinygo-org/tinygo/pull/4467) in tinygo.

### CODE REVIEW

File: content/docs/reference/usage/subcommands.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ description: |
 TinyGo tries to be similar to the main `go` command in usage. It consists of the following main subcommands:
 
 ### build
-Compile the given program. The output binary is specified using the ``-o``
-parameter. The generated file type depends on the extension:
+Build compiles the packages named by the import paths, along with their dependencies, but it does not install the results.
+The generated file type depends on the extension:
```
Comment (aykevl) [inline content/docs/reference/usage/subcommands.md]: Nit: right now TinyGo only supports a single import path.
```suggestion
Build compiles the package named by the import path, along with its dependencies, but it does not install the result.
The generated file type depends on the extension:
```

**Comment by leongross** [comment]
@deadprogram

**Comment by leongross** [comment]
@aykevl

**Comment by leongross** [comment]
@deadprogram @aykevl @dgryski

**Comment by deadprogram** [comment]
Merging, thanks @leongross

**Pull request merged**